.travis.yml

```diff
@@ -20,7 +20,8 @@ sudo: false
 
 install: travis_retry composer require "wikibase/data-model=$DM" --prefer-source
 
-script: composer ci
+# Hack: Don't run phpmd with HHVM
+script: if $(php --version | grep -q HipHop); then composer test; else composer ci; fi
```

Comment:
would phpenv version-name work here as well? (no big deal, just mentioning as it is used 3 lines below)